.gitmodules

```diff
@@ -12,6 +12,7 @@
 	path = ghcide
 	# url = https://github.com/alanz/ghcide.git
 	# url = https://github.com/wz1000/ghcide.git
-	url = https://github.com/haskell/ghcide.git
+	# url = https://github.com/haskell/ghcide.git
```

Comment:
What's going on here? Accident?

Reply:
No, i have to use my own branch of ghcide (see https://github.com/haskell/ghcide/pull/731) which takes in account the client setting change. I'll switch to upstream ghcide after merging it